integration/simple-prototype-defaults/simple-test.ts

```diff
@@ -0,0 +1,22 @@
+import { Child_Type, Simple, StateEnum } from './simple';
+
+describe('simple', () => {
+  it('generates types correctly', () => {
```

Comment:
Maybe add a test that does a `Simple.decode` that shows a from-default value like `age: 0` has `Object.keys().includes("age") === false` or what not?